ros_buildfarm/templates/snippet/scm_git.xml.em

```diff
@@ -6,6 +6,9 @@
         <refspec>@ESCAPE(refspec)</refspec>
 @[end if]@
         <url>@ESCAPE(url)</url>
+@[if vars().get('git_ssh_credential_id')]@
```

Comment:
Why is the condition so complicated? Shouldn't the following be enough:

    if git_ssh_credential_id